From 1a5364f299f2f7b42ed8b01635451dfb2184a525 Mon Sep 17 00:00:00 2001
From: Wu Gao <wugao@codeaurora.org>
Date: Tue, 20 Oct 2020 03:50:51 -0700
Subject: [PATCH] dpp: Fix compilation issue about ctrl_iface.c

It reports that no "dpp.h" when compiling ctrl_iface.c.
The fix is including "common/dpp.h"

Change-Id: I6c3ff2514a87f96875c75024f543738292625ff6
Signed-off-by: Wu Gao <wugao@codeaurora.org>
---
 wpa_supplicant/dpp_supplicant.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wpa_supplicant/dpp_supplicant.h b/wpa_supplicant/dpp_supplicant.h
index 2a5c8c6..82d0e88 100644
--- a/wpa_supplicant/dpp_supplicant.h
+++ b/wpa_supplicant/dpp_supplicant.h
@@ -10,7 +10,7 @@
 #ifndef DPP_SUPPLICANT_H
 #define DPP_SUPPLICANT_H
 
-#include"dpp.h"
+#include"common/dpp.h"
 
 enum dpp_status_error;
 
-- 
2.7.4

